packages/astro/src/core/errors/dev/utils.ts

```diff
@@ -32,10 +33,15 @@ export function collectErrorMetadata(e: any, rootFolder?: URL | undefined): Erro
 		// - We'll fail to show the file's content in the browser
 		// - We'll fail to show the code frame in the terminal
 		// - The "Open in Editor" button won't work
+
+		// Normalize the paths so that we can correctly detect if it's absolute on any platform
+		const normalizedFile = error.loc?.file?.replace(/\\/g, '/');
```

Comment:
Normalize the separators because if we are on Windows the path sometimes uses `/` (if the error is in a component) and sometimes uses `\\` (if the error is in a page).

Reply:
@Princesseuh Thank you so much for pointing that out! Pushed a fix.